app/templates/adIndex.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Teacher Registration</title>
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<script  src="https://code.jquery.com/jquery-3.2.1.js" integrity="sha256-DZAnKJ/6XZ9si04Hgrsxu/8s717jcIzLy3oi35EouyE="  crossorigin="anonymous"></script>
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+
+<link href="https://necolas.github.io/normalize.css/7.0.0/normalize.css" rel="stylesheet">
+
+</head>
+<body>
+<div class="container col-md-6 col-md-offset-3">
+<div class="panel panel-primary">
+  <div class="panel-heading">
+    <h3 class="panel-title">Submit Ad</h3>
+  </div>
+  <div class="panel-body">
+  <form  action="thankyou.html" class="form-horizontal">
+  <fieldset>
+    <legend>Job Requirement</legend>
+    <div class="form-group">
+      <label for="inputSchoolName" class="col-md-2 control-label">School Name</label>
+      <div class="col-md-6">
+        <input class="form-control" id="inputSchoolName" placeholder="School Name" type="text" required>
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label for="inputJobTitle" class="col-md-2 control-label">Job Title</label>
+      <div class="col-md-6">
+        <input class="form-control" id="inputJobTitle" placeholder="Job Title" type="text" required>
+      </div>
+    </div>
+
+    <!--div class="form-group">
+      <label for="inputSubject" class="col-md-2 control-label">Subject</label>
+      <div class="col-md-6">
+        <input class="form-control" id="inputSubject" placeholder="Subject" type="text">
+      </div>
+    </div-->
+
+    <div class="form-group">
+      <label for="inputSubject" class="col-md-2 control-label" required>Subject</label>
+      <div class="col-md-6">
+        <select class="form-control" id="inputSubject" required>
+          <option>SELECT SUBJECT</option>
+          <option>MATHEMATICS</option>
+          <option>ENGLISH LANGUAGE</option>
+          <option>BIOLOGY</option>
+          <option>ECONOMINS</option>
+        </select>
+     </div>
+    </div>
+
+
+    <!--div class="form-group">
```

Comment:
Please delete commented out code. If you ever need the code, you can get it back from version control. 

Please read the section 'Commented-Out Code' in the book 'Clean Code' here -
 https://books.google.co.uk/books?id=_i6bDeoCQzsC&lpg=PP1&dq=clean%20code&pg=PA68#v=onepage&q&f=false